### PR TITLE
Call unregister sw in index

### PR DESCRIPTION
**Pull request by dwalintukan**
Call unregister for existing service workers to remove any caching that was previously registered in earlier versions.

### CODE REVIEW

File: src/registerServiceWorker.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ export default function register() {
       navigator.serviceWorker
         .register(swUrl)
         .then((registration) => {
+          console.debug('Registered service worker', registration);
```
Comment (frankobe) [inline src/registerServiceWorker.js]: remove log?

File: src/registerServiceWorker.js [hunk 2]
```diff
@@ -45,6 +47,7 @@ export function unregister() {
   if ('serviceWorker' in navigator) {
     navigator.serviceWorker.ready.then((registration) => {
       registration.unregister();
+      console.debug('Unregistered service worker');
```
Comment (frankobe) [inline src/registerServiceWorker.js]: remove log?

File: src/registerServiceWorker.js (hunk 1 above)
Comment (dwalintukan) [reply src/registerServiceWorker.js]: guess not necessary. ok ill remove

File: src/registerServiceWorker.js (hunk 2 above)
Comment (dwalintukan) [reply src/registerServiceWorker.js]: removed

**Pull request merged**